drivers/gpio/Kconfig.sifive

```diff
@@ -10,135 +10,3 @@ menuconfig GPIO_SIFIVE
 	  Enable driver for the SiFive Freedom GPIO controller.
 
 	  Says n if not sure.
-
```

Comment:
Above change menuconfig to config to eliminate the compliance problem.

Reply:
Thanks! Fixed it.